deploy/packages/deb/debian/postinst

```diff
@@ -38,6 +35,12 @@ chmod -R 0755 /usr/lib/emqx/bin
 ln -s /usr/lib/emqx/bin/emqx /usr/bin/emqx
 ln -s /usr/lib/emqx/bin/emqx_ctl /usr/bin/emqx_ctl
 
+if pidof systemd; then
```

Comment:
there is no `systemd` process in my ubuntu 20.
but other processes like `systemd-journald` etc.

is this to check if systemd is available? maybe use `commnad -v systemctl` ?

Reply:
I have ubuntu 20.04 LTS too. Could you run `pidof systemd` ?

It is to check if the systemd is in use as the init process.

I want to ensure that systemd command is available and also in-use. because in some env, like container , systemd command tool is available but not started.

Reply:
`pidof systemd` exists with `1` in my case (systemd is running, `systemctl status` command prints out service statuses)

Reply:
I changed to use `systemctl status` now.